exercises/reverse-string/canonical-schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "exercise": "reverse-string",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "detects reverse-string",
+      "property": "reverse-string",
+      "input": "robot",
+      "expected": "tobor"
+    },
+    {
+      "description": "detects empty string",
+      "property": "reverse-string",
+      "input": "",
+      "expected": "String is empty, input valid string"
```

Comment:
the reverse of `""` should be `""`. This string is only expected if the input is `"gnirts dilav tupni ,ytpme si gnirtS"`